agot-bg-game-server/src/common/cancelled-game-state/CancelledGameState.ts

```diff
@@ -9,7 +9,7 @@ export default class CancelledGameState extends GameState<EntireGame> {
     }
 
     onClientMessage(_user: User, _message: ClientMessage): void {
-        
+        return;
```

Comment:
I'll turn off the linter message for empty function, since it's a pattern that we use frequently in the codebase, so no need to add `return;` here.

Reply:
I found that line as a trailing whitespace crept in again. I always try to do a \s$ search before PRing... But sure. I removed the commit.